.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: java
 sudo: false
+
 cache:
   directories:
   - .autoconf
   - $HOME/.m2
+  - $HOME/.zinc
+
+script: mvn scoverage:report
```

Comment:
Yea, that's intentional, though maybe there's a clearer way to do it, and/or I can document things better. Any ideas?

Maybe I'll try leaving `script` as the default `mvn test -B`, and doing:

```yml
after_success:
- mvn scoverage:report -DskipTests coveralls:report
```

or something like that.

By default `mvn scoverage:report` runs the tests.